aca_preview/preview.py

```diff
@@ -520,7 +522,7 @@ def check_catalog(self):
 
         self.check_guide_geometry()
         self.check_acq_p2()
-        self.check_bright_guide_for_ers()
+        self.check_guide_fraction()
```

Comment:
Try to maintain consistency with existing conventions.  In this case we always use "guide_count".  For me `guide_fraction` is a fraction like 0.77.

Reply:
For me, sticking to convention here was actually confusing.  This is not the guide_count of starcheck or the count of the number of guide stars in the catalog.  I'd do better with a new name, but OK.